lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -167,6 +167,9 @@ bool CPUBackend::isOpSupported(const NodeInfo &NI) const {
   case Kinded::Kind::ExpNodeKind:
     return NI.allInputsAndOutputsHaveSameElemKind({ElemKind::FloatTy});
 
+  case Kinded::Kind::ModuloNodeKind:
+    return NI.allInputsAndOutputsHaveSameElemKind({ElemKind::Int64ITy});
```

Comment:
I think we could support Mod for `Int32ITy` simply by adding `_i32` versions of the kernels.  Is that so?

Reply:
I am not sure we should, the Mod operator patch defines the inputs as int64_t https://github.com/pytorch/glow/pull/2345/commits/5bf85214523684440268a6befc9cbc95d77b4ffb

Perhaps @jackm321 can weigh in on this?

Reply:
@SameerAsal check the Interpreter.cpp for the corresponding op support. I think we can do the same here.

Reply:
OK. I was just concerned because ion our test cases we only added tensors of type int64_t. I had a chat with bert and greed to support both and add a tests for int32_t.

Thanks!